src/corelocation.cs

```diff
@@ -871,4 +890,19 @@ interface CLBeaconIdentityConstraint : NSCopying, NSSecureCoding {
 		[BindAs (typeof (short?))]
 		NSNumber Minor { get; }
 	}
+
+	[Model]
```

Comment:
It's unclear how this protocol is supposed to be used, if it's a *Delegate type somewhere or not (it doesn't seem to show up anywhere else in the API).

In the meantime, I think we can remove the `[Model]` attribute, just in case it's not a *Delegate type. It's easier to add it later than to remove it.

Reply:
I added model due to the documentation from the method that the protocol has:

```
+// Called just before this extension is terminated by the system.
```

But I'll remove, since it can be re-added.